src/main/java/com/internet/shop/db/Storage.java

```diff
@@ -1,16 +1,43 @@
 package com.internet.shop.db;
 
+import com.internet.shop.model.Order;
 import com.internet.shop.model.Product;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.model.User;
 import java.util.ArrayList;
 import java.util.List;
 
 public class Storage {
     public static final List<Product> products = new ArrayList<>();
+    public static final List<Order> orders = new ArrayList<>();
+    public static final List<ShoppingCart> shoppingCarts = new ArrayList<>();
+    public static final List<User> users = new ArrayList<>();
     private static Long productId = 0L;
+    private static Long orderId = 0L;
+    private static Long shoppingCardId = 0L;
+    private static Long userId = 0L;
 
     public static void addProduct(Product product) {
         productId++;
         product.setId(productId);
         products.add(product);
     }
+
+    public static void addOrder(Order order) {
+        orderId++;
+        order.setId(orderId);
```

Comment:
```suggestion
        order.setId(++orderId);
```
same in other methods

Reply:
Done